.gitlab-ci.yml

```diff
@@ -76,4 +79,7 @@ production:
     K8S_SECRET_NOTIFICATION_SERVICE_API_TOKEN: "$GL_PRODUCTION_NS_API_TOKEN"
     K8S_SECRET_NOTIFICATION_SERVICE_API_URL: "$GL_PRODUCTION_NS_API_URL"
     K8S_SECRET_CAPTCHA_ENABLED: "$GL_PRODUCTION_CAPTCHA_ENABLED"
-    K8S_SECRET_RECAPTCHA_SECRET_KEY: "$GL_PRODUCTION_RECAPTCHA_SECRET_KEY"
\ No newline at end of file
+    K8S_SECRET_RECAPTCHA_SECRET_KEY: "$GL_PRODUCTION_RECAPTCHA_SECRET_KEY"
+    K8S_SECRET_KEYWORD_SET_CATEGORY_ID: "$GL_PRODUCTION_KEYWORD_SET_CATEGORY_ID"
+    K8S_SECRET_KEYWORD_SET_ADDITIONAL_CRITERIA_ID: "$GL_PRODUCTION_KEYWORD_SET_CATEGORY_ID"
+    K8S_SECRET_KEYWORD_SET_TARGET_GROUP_ID: "$GL_PRODUCTION_KEYWORD_SET_CATEGORY_ID"
```

Comment:
Are these all the same on purpose?

Reply:
No, they are the same because I forgot to change them, fixed :)